### PR TITLE
Fix/kn/home page styles improvement

### LINKED ISSUE #116: Home Page in Desktop mode doesn't looks like the design
**Describe the bug**
The home page doesn't looks how the design

**To Reproduce**
Go to https://dev--azordev-dasher-user.netlify.app/

**Expected behavior**
I have listed the bugs here

- [ ] 1. The **Dasher Guy** image should be in the bottom of the page and not at start. I have see that exists an effort for move to up the image (`margin-top: -23%`). Maybe this style of margin-top is for the mobile and we forgot to remove it on desktop. (Please check screenshot 1)

- [ ] 2. The "Hola!" text should be more big, in Mobile looks good, but in desktop is not according to the design. (Please check screenshot 2). Please add a [media query](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) to increment the font size in desktop.

- [ ] 3. At desktop device, the "Hola!" text and their paragraph should be aligned to the left (Please check image 3). Please change `text-align: center` for `text-align: left` and add margin / padding to separate it for the borders

- [ ] 4. The "Hola!" text and the button "Aceptar" have very much space between. This can be solved? (Please check image 4)

**Screenshots**
1. Dasher Guy is at the start of the page instead of the bottom
**Actually is:**
imagen
**Expected:** (Note that the guy is at the left-bottom corner of the page, and not have margin bottom or left to separate it from the borders)
imagen
I have notice this effort for move up the image, can be an error?:
imagen
2. "Hola" text is some little in desktop 
**Actually is:**
imagen | 
**Expected:**
imagen |
3. "Hola!" text and paragraph should be aligned to the left
**Actually is:**
imagen
**Expected:**
imagen
4. The button "aceptar" is very separated of the rest
imagen |

### PR DESCRIPTION
**Pull request by Saikilito**
### Description
Improve styles for web and mobile into home page

fixes # (issue)
- #116 
- 
#### Screenshots

**dev branch**
image

image

**current branch** 
image

image

#### Type of change

Improve styles

- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
Just one detail, when in large mobile sizes, like iPhoneX, content is not centered
image

Can you center vertically, please?

File: src/pages/Home/Home.layout.jsx [hunk 1]
```diff
@@ -16,13 +17,24 @@ import { Button, Header, Logo, ManImage, Text } from './Home.styled'
...
   <Header>
-    <Logo src={logo} alt="logo" />
-    <Text as="h1" color="secondary" large>
-      {title}
-    </Text>
-    <Text small>{subtitle}</Text>
-    <ManImage src={deliveryMan} alt="Delivery man" />
-    <Button to="/check">{buttonText}</Button>
+    <HalfSection bg={Colors.yellow06}>
+      {/**/}
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.layout.jsx]: These spaces are not necessary, can you try to remove them?

File: src/pages/Home/Home.layout.jsx [hunk 2]
```diff
@@ -16,13 +17,24 @@ import { Button, Header, Logo, ManImage, Text } from './Home.styled'
...
+      <Logo src={logo} alt="logo" />
+      <ManImage style={{ display: 'nones' }} src={deliveryMan} alt="Delivery man" />
+      {/**/}
+    </HalfSection>
+
+    <HalfSection mobileHeight="400px" bg={Colors.blue08} fxJtyCt="center" fxAlIt="center">
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.layout.jsx]: The props names "fxJtyCt" and "fxAlIt" can be better, please change them, currently is hard to follow that they meant, even after looking at the CSS component

Comment (Israel-Laguan) [reply src/pages/Home/Home.layout.jsx]: Also there are not other places these props changes, can you try using defaults?

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]

File: src/pages/Home/Home.layout.jsx (hunk 1 above)
Comment (Saikilito) [reply src/pages/Home/Home.layout.jsx]: ready

File: src/pages/Home/Home.layout.jsx (hunk 2 above)
Comment (Saikilito) [reply src/pages/Home/Home.layout.jsx]: props names changed

**Comment by Saikilito** [comment]
@Israel-Laguan  Iphone X updated

image

File: src/pages/Home/Home.styled.jsx [hunk 3]
```diff
@@ -51,20 +119,31 @@ export const Button = styled(Link)`
...
+    border-radius: 2rem;
+    letter-spacing: 2px;
+    transform: translate(0, 0);
   }
-`
 
-export const Logo = styled.img`
-  display: none;
+  @media (min-height: 700px) and (min-width: 375px) and (max-width: 600px) {
+    transform: translate(0%, 1200%);
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: WFT with those values? 😮

File: src/pages/Home/Home.styled.jsx [hunk 4]
```diff
@@ -51,20 +119,31 @@ export const Button = styled(Link)`
...
+  }
 
-  @media (min-width: 960px) {
-    margin-left: -94%;
-    display: block;
-    margin-top: 10%;
+  @media (max-width: 320px) {
+    transform: translate(0%, 900%);
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: WFT with those values? 😮

File: src/pages/Home/Home.styled.jsx [hunk 5]
```diff
@@ -51,20 +119,31 @@ export const Button = styled(Link)`
   text-align: center;
   text-decoration: none;
   line-height: 40px;
+  font-weight: bold;
+
+  position: relative;
+  transform: translate(0%, 1100%);
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx:164]: WFT with those values? 😮

File: src/pages/Home/Home.styled.jsx [hunk 6]
```diff
@@ -3,46 +3,114 @@ import { Link } from 'react-router-dom'
...
+    margin: 10px 0 30px -5px;
   }
 `
 
+export const SubtitleText = styled(Text)`
+  margin-top: 3px;
+  font-size: 1rem;
+
+  @media (min-width: 960px) {
+    margin: 15px 0 15px 0;
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: what about

```css
margin: 15px 0;
```

File: src/pages/Home/Home.styled.jsx [hunk 7]
```diff
@@ -3,46 +3,114 @@ import { Link } from 'react-router-dom'
...
-  height: 65vh;
+  width: 70%;
+  margin-top: 20%;
+
+  object-fit: contain;
+  position: absolute;
+  transform: translateY(-80%);
+
+  @media (min-width: 960px) {
+    transform: translateY(0%);
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: Did you tried `transform: none;`, `transform: unset;` or `transform: revert;`?

File: src/pages/Home/Home.styled.jsx [hunk 8]
```diff
@@ -3,46 +3,114 @@ import { Link } from 'react-router-dom'
...
+  @media (max-width: 320px) {
+    margin-top: 0;
+  }
+`
+export const TextContainer = styled.div`
+  transform: translateY(-80%);
+  max-width: 100%;
+
+  @media (min-width: 960px) {
+    transform: translateY(0%);
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: Did you tried `transform: none;`, `transform: unset;` or `transform: revert;`?

File: src/pages/Home/Home.styled.jsx [hunk 9]
```diff
@@ -51,20 +119,31 @@ export const Button = styled(Link)`
...
-    width: 45%;
+    height: 50%;
+    width: 100%;
+    padding: 1rem 0 3rem 0;
+    font-weight: bold;
+    font-size: 1.5rem;
+
+    border-radius: 2rem;
+    letter-spacing: 2px;
+    transform: translate(0, 0);
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: Did you tried `transform: none;`, `transform: unset;` or `transform: revert;`?

File: src/pages/Home/Home.styled.jsx [hunk 10]
```diff
@@ -3,46 +3,114 @@ import { Link } from 'react-router-dom'
...
+`
+export const HalfSection = styled.div`
+  ${props => `background-color: ${props.backgroundColor};`}
+  ${props => `justify-content: ${props.justifyContent};`}
+  ${props => `align-items: ${props.alignItems};`}
+  ${props => `height: ${props.mobileHeight};`}
+
+  display: flex;
+  flex-direction: column;
+  @media (min-width: 960px) {
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: space before `@media` please

File: src/pages/Home/Home.layout.jsx [hunk 11]
```diff
@@ -16,13 +17,20 @@ import { Button, Header, Logo, ManImage, Text } from './Home.styled'
...
-    <Logo src={logo} alt="logo" />
-    <Text as="h1" color="secondary" large>
-      {title}
-    </Text>
-    <Text small>{subtitle}</Text>
-    <ManImage src={deliveryMan} alt="Delivery man" />
-    <Button to="/check">{buttonText}</Button>
+    <HalfSection backgroundColor={Colors.yellow06}>
+      <Logo src={logo} alt="logo" />
+      <ManImage style={{ display: 'nones' }} src={deliveryMan} alt="Delivery man" />
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.layout.jsx]: `display: 'nones'` shouldn't be valid, maybe you meant `display: 'none;'`, alsy why you wanted to hide this?

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]

File: src/pages/Home/Home.styled.jsx (hunk 6 above)
Comment (Saikilito) [reply src/pages/Home/Home.styled.jsx]: this does not add any value

File: src/pages/Home/Home.styled.jsx (hunk 3 above)
Comment (Saikilito) [reply src/pages/Home/Home.styled.jsx]: this works with the absolute position and the flex reverse

File: src/pages/Home/Home.styled.jsx (hunk 4 above)
Comment (Saikilito) [reply src/pages/Home/Home.styled.jsx]: this works with the absolute position and the flex reverse

File: src/pages/Home/Home.styled.jsx (hunk 5 above)
Comment (Saikilito) [reply src/pages/Home/Home.styled.jsx:164]: this works with the absolute position and the flex reverse

File: src/pages/Home/Home.layout.jsx (hunk 11 above)
Comment (Saikilito) [reply src/pages/Home/Home.layout.jsx]: ready

File: src/pages/Home/Home.styled.jsx (hunk 8 above)
Comment (Saikilito) [reply src/pages/Home/Home.styled.jsx]: ready

File: src/pages/Home/Home.styled.jsx (hunk 7 above)
Comment (Saikilito) [reply src/pages/Home/Home.styled.jsx]: ready

File: src/pages/Home/Home.styled.jsx (hunk 9 above)
Comment (Saikilito) [reply src/pages/Home/Home.styled.jsx]: ready

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
In tablet mode it looks weird :(
image

**Comment by Israel-Laguan** [comment]
Good work @Saikilito !
